BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-servicing-02012-01
\ No newline at end of file
+2.0.0-servicing-02201-03
```

Comment:
Latest version which includes the parallel download of files from Azure is 2.0.0-servicing-02213-03